coldfront/core/user/forms.py

```diff
@@ -164,6 +176,38 @@ class UserProfileUpdateForm(forms.Form):
         help_text='The number must be in E.164 format (e.g. +12125552368).',
         label='Phone Number', required=False)
 
+    def clean_phone_number(self):
+        cleaned_data = super().clean()
+        phone_number = cleaned_data.pop('phone_number', '')
+        return phone_number
+
+    def clean_first_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('first_name', '')
+        return name.title()
+
+    def clean_middle_name(self):
+        cleaned_data = super().clean()
+        name = cleaned_data.pop('middle_name', '')
+        return name.title()
+
+    def clean_last_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('last_name', '')
+        return name.title()
+
+    def save(self, commit=True):
+        model = super().save(commit=False)
+        model.username = model.email
+        model.is_active = False
```

Comment:
This came from the above `save` method, but is intended for use in registration only.

For context, when a user first registers to the portal, we set them to inactive (`is_active = False`). After they verify their email address, some other view will set their account to active so that they can actually log in.

If we were to add this statement here, updating one's profile would also deactivate the user's account, locking them out.